card/mod/05_email/lib/card/follow_option.rb

```diff
@@ -48,6 +50,15 @@ def follow_opts args
         add_option args, :main
       end
       
+      def follow_test opts={}, &block
+        codename = get_codename opts
+        Card::FollowOption.test[codename] = block
+      end
+      
+      def follow_test_option option, &block
+        Card::FollowOption.test_option[option] = block
+      end
+      
```

Comment:
the wording here obscures the functionality. `follow_test_option` is not about some optional options it's the core of finding the followers (took me a while to get that). I don't think we need in `follow_test_options` anything else but user ids. Maybe something in this direction:
````ruby
def select_follower_ids &block
  Card::FollowOption.select_follower_ids[get_codename({})] = block
end
````
and then you use it as follows in edited.rb for example:
````ruby
self.select_follower_ids do |card|
  Card.search( :editor_of=>card.name, :return=>:id ).map &:to_i
end
self.follow_test do |user_id, approved_user_ids|  
  approved_user_ids.include? user_id
  # this could be the default and we override it for *always and *never
end